examples/plot_parcellation.py

```diff
@@ -14,7 +14,10 @@
 subject_id = 'fsaverage'
 hemi = 'both'
 surf = 'inflated'
-view = 'frontal'
+if 'patch' in surf.lower().split('.'):
+    view=None
+else:
+    view = 'frontal'
```

Comment:
None of these conditionals are needed with the `show_view` modification suggested above.